docs/concepts/reactiveui/reactive-view-model.md

```diff
@@ -59,3 +59,65 @@ Any change to the view model description property is achieved using the `set` ac
 When _Avalonia UI_ uses the binding to **Update** the view model, the `set` accessor ensures that any parts of the view model that depend on the description property can also react to the change if necessary.
 
 On the next page, you will learn how a reactive command acts as a special case of the view model update.
+
+## Performing Data Validation
+ReactiveUI provides the ability to perform data validation from the ViewModel that integrates well with the Avalonia fluent controls. The example shows how this can be done. The example is for a ViewModel that exposes a directory path string that is bounded to a TextBox control on the View.
```

Comment:
A bit imprecise to say it integrates with the Avalonia Fluent controls as it will work for any theme. It would integrate with one of Avalonia's data validation plugins: either Data Annotations, `INotifyDataErrorInfo`, or via exceptions. I'm not sure which one ReactiveUI is using here.